examples/computer_vision/efficientdet_pytorch/model_def.py

```diff
@@ -0,0 +1,394 @@
+                sampler=sampler,
+                pin_memory=pin_mem,
+                collate_fn=collate_fn,
+            )
+
+            return loader
+
+    def build_training_data_loader(self):
+
+        dataset_train, self.dataset_eval = create_dataset(self.args.dataset, self.args.root)
+
+        self.labeler = None
+        if not self.args.bench_labeler:
+            self.labeler = AnchorLabeler(Anchors.from_config(self.model_config), self.model_config.num_classes, match_threshold=0.5)
+
+        loader_train = self._create_loader(
+            dataset_train,
+            input_size=self.input_config['input_size'],
+            batch_size=self.context.get_per_slot_batch_size(),
+            is_training=True,
+            use_prefetcher=self.args.prefetcher,
+            re_prob=self.args.reprob,
+            re_mode=self.args.remode,
+            re_count=self.args.recount,
+            # color_jitter=self.args.color_jitter,
+            # auto_augment=self.args.aa,
+            interpolation=self.args.train_interpolation or self.input_config['interpolation'],
+            fill_color=self.input_config['fill_color'],
+            mean=self.input_config['mean'],
+            std=self.input_config['std'],
+            num_workers=self.args.workers,
+            distributed=self.args.distributed,
+            pin_mem=self.args.pin_mem,
+            anchor_labeler=self.labeler,
+        )
+        if self.model_config.num_classes < loader_train.dataset.parser.max_label:
+            logging.error(
+                f'Model {self.model_config.num_classes} has fewer classes than dataset {loader_train.dataset.parser.max_label}.')
+            exit(1)
+        if self.model_config.num_classes > loader_train.dataset.parser.max_label:
+            logging.warning(
+                f'Model {self.model_config.num_classes} has more classes than dataset {loader_train.dataset.parser.max_label}.')
+
+        self.data_length = len(loader_train)
+
+        return loader_train
+
+
+    def build_validation_data_loader(self):
+        if self.args.val_skip > 1:
+            self.dataset_eval = SkipSubset(self.dataset_eval, self.args.val_skip)
+        self.loader_eval = self._create_loader(
+            self.dataset_eval,
+            input_size=self.input_config['input_size'],
+            batch_size=self.context.get_per_slot_batch_size(),
+            is_training=False,
+            use_prefetcher=self.args.prefetcher,
+            interpolation=self.input_config['interpolation'],
+            fill_color=self.input_config['fill_color'],
+            mean=self.input_config['mean'],
+            std=self.input_config['std'],
+            num_workers=self.args.workers,
+            distributed=self.args.distributed,
+            pin_mem=self.args.pin_mem,
+            anchor_labeler=self.labeler,
+        )
+        
+        self.evaluator = self.create_evaluator(self.args.dataset, self.loader_eval.dataset, pred_yxyx=False)
+
+        self.eval_counter = 0
```

Comment:
This is still here, but the comment got resolved.

Reply:
github did github things on merged, should be removed now.